yarn.lock

```diff
@@ -13033,6 +13038,13 @@ d3-array@2, d3-array@2.12.1, d3-array@^2.3.0:
   dependencies:
     internmap "^1.0.0"
 
+d3-array@3.2.2, d3-array@^3.2.2:
```

Comment:
__nit:__ we can merge `d3-array@3.2.2, d3-array@^3.2.2:` and `"d3-array@1 - 3", "d3-array@2 - 3", "d3-array@2.10.0 - 3", "d3-array@2.5.0 - 3", d3-array@^3.1.1:` above - just remove these `d3-array` declarations and re-run `yarn kbn bootstrap` - yarn will fix merge everything it can merge (take a look at how Yarn handled it in my PR: https://github.com/elastic/kibana/pull/152700/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de).